ddtrace/internal/telemetry/writer.py

```diff
@@ -216,11 +225,13 @@ def _create_telemetry_request(self, payload, payload_type, sequence_id):
             "request_type": payload_type,
         }
 
-    def _restart(self):
+    def _fork_writer(self):
         # type: () -> None
-        if self.status == ServiceStatus.RUNNING:
-            self.disable()
-        self.enable()
```

Comment:
We still need to restart the writer since post-fork Python kills all the threads.